server_admin/topics/authentication/webauthn.adoc

```diff
@@ -244,3 +244,52 @@ The final configuration of the flow looks similar to this:
+
+[[_webauthn_loginless]]
+==== LoginLess WebAuthn
+
+{project_name} uses WebAuthn for two-factor authentication, but you can use WebAuthn as the first-factor authentication. In this case, users with `passwordless` WebAuthn credentials can authenticate to {project_name} without submitting a login nor a password. {project_name} can use WebAuthn as both the loginless/passwordless and two-factor authentication mechanism in the context of a realm.
+
+An administrator typically requires that Security Keys registered by users for the WebAuthn loginless authentication meet different requirements. Loginless authentication requires users to authenticate to the security key (eg. by using a PIN code or a fingerprint) and that the cryptographic keys associated with the loginless credential are stored physically on the securitey key (not all security keys meet that kind of requirements, please check with your security key vendor if your device supports 'user verification' and 'resident key', see <<_webauthn-supported-keys, Supported Security Keys>>)
+
+{project_name} permits administrators to configure the `WebAuthn Passwordless Policy` in a way that allows loginless authentication. Please note that loginless authentication can only be configured with `WebAuthn Passwordless Policy` and with `WebAuthn Passwordless` credentials. WebAuthn loginless authentication and WebAuthn passwordless authentication can be configured on the same realm but will share the same policy `WebAuthn Passwordless Policy`.
+
+===== Setup
+
+Set up WebAuthn Loginless support as follows:
+
+. Register a new required action for WebAuthn passwordless support. Use the steps described in <<_webauthn-register, Enable WebAuthn Authenticator Registration>>. Register the `Webauthn Register Passwordless` action.
+
+. Configure the `WebAuthn Passwordless Policy` . Perform the configuration in the Admin Console, `Authentication` section,  in the tab `WebAuthn Passwordless Policy`. You have to set the *User Verification Requirement* to *Required* and the *Require Resident Key* to *Yes* when you configure the policy for loginless scenario. Please note that since there isn't a dedicated Loginless policy it won't be possible to mix authentication scenarios with user verification=no/resident key=no and loginless scenarios (user verification=yes/resident key=yes). Please also note that storage capacity is usually very limited on security keys meaning that you won't be able to store many resident keys on your security key.
+
+. Configure the authentication flow. Create a new authentication flow, add the "WebAuthn Passwordless" execution and set the Requirement setting of the execution to `Required`
+
+The final configuration of the flow looks similar to this:
+
```

Comment:
.LoginLess flow
image:images/webauthn-loginless-flow.png[LoginLess flow]

Reply:
Again, this is coming from the existing "Passwordless WebAuthn together with Two-Factor" section which doesn't have the ".PasswordLess flow" header. Should I fix both or none ?

Reply:
.LoginLess is just a heading for the image 
[LoginLess flow] is ALT text, but neither is essential if it is inconsistent.

Reply:
Thanks. Resolved. I also updated the passwordless section to be consistent